src/usr/local/www/services_ntpd.php

```diff
@@ -59,6 +60,10 @@
 	unset($input_errors);
 	$pconfig = $_POST;
 
+	if (!is_numericint($_POST['ntpmaxpeers']) || ($_POST['ntpmaxpeers'] < 4) || ($_POST['ntpmaxpeers'] > $max_candidate_peers)) {
```

Comment:
`4` should be in its own similar variable, like `$min_candidate_peers` -- here and in the option text. Same reason as max was, so it could be changed later if need be.

Also please break up this long line -- After the `||`, start a new line and intent it the same level as the `if` + four spaces. (https://docs.netgate.com/pfsense/en/latest/development/developer-style-guide.html#indent-style)